keyboards/crkbd/keymaps/kidbrazil/README.md

```diff
@@ -45,8 +45,19 @@ customize this with showing layer and USB information. I also tried my best to
 get a dormant / sleep state going but it is hit or miss and often only works on
 the master hand.
 
+## OLED & RGB Matrix timeout
+This keymap will set a automated timeout system for the OLED screen and the RGB
+matrix. After 3 minutes or so the LED screen will display the logo on both
+halves and 5 minutes after that both the LED and the Matrix will be switched
+off.
+
+Once a user hits the keys again, the LED matrix will turn back on unless the
+user has disabled it via RGB_TOG.
+
+## Flashing
+To flash this on your CRKBD simply use the `make crkbd:kidbrazil:avrdude`
```

Comment:
The `:flash` suffix works with all our most common bootloaders and will call the correct program based on the `BOOTLOADER` rule from `rules.mk`.

```suggestion
To flash this on your CRKBD simply use the `make crkbd:kidbrazil:flash`
```

Reply:
Ohh ok was not aware of that. Thank you very much!